androidApp/src/androidTest/java/com/mbta/tid/mbta_app/android/map/HomeMapViewTests.kt

```diff
@@ -294,44 +302,58 @@ class HomeMapViewTests {
             .assertDoesNotExist()
     }
 
-    @OptIn(ExperimentalTestApi::class)
     @Test
-    @Ignore("flaky test passing locally but failing in CI")
-    fun testOverviewShownOnStopDetails(): Unit = runBlocking {
+    fun testRecenterButtonVisibilityCalledWhenOnStopDetails(): Unit = runBlocking {
         val locationManager = MockLocationDataManager()
         locationManager.hasPermission = true
-        val viewModel =
-            MapViewModel(ConfigUseCase(MockConfigRepository(), MockSentryRepository()), {})
 
-        viewModel.setGlobalResponse(GlobalResponse(objects = TestData))
+        val searchVM =
+            SearchResultsViewModel(
+                MockAnalytics(),
+                MockSearchResultRepository(),
+                VisitHistoryUsecase(MockVisitHistoryRepository()),
+            )
         val viewportProvider = ViewportProvider(MapViewportState())
-        viewModel.loadConfig()
+
+        var updateCenterButtonVisibilityCalled = false
+
+        AutofillProvider.forMockMode.types.register(StateFlow::class) { MutableStateFlow(null) }
+
+        AutofillProvider.forMockMode.types.register(StateFlow::class) { MutableStateFlow(null) }
+
+        AutofillProvider.forMockMode.types.register(Flow::class) { MutableStateFlow(null) }
+
+        val mapVM = mock<IMapViewModel>(MockMode.autofill)
+        every { mapVM.selectedStop } returns MutableStateFlow(TestData.getStop("121"))
+        every { mapVM.configLoadAttempted } returns MutableStateFlow(true)
+        every { mapVM.showRecenterButton } returns MutableStateFlow(false)
+        every { mapVM.showTripCenterButton } returns MutableStateFlow(false)
```

Comment:
This may have been sufficiently easy to do without mokkery, but I wanted to work with it as a tool that exists already in our test code but is maybe underutilized. Would be interested in feedback on whether we'd want to adopt mokkery more widely as a tool for mocking view models in UI tests, and the extent to which that is still relevant in a world where all VMs are using molecule. (I _think_ it is still relevant, but need to look closer)